src/main/java/pantrypal/inventory/IngredientInventory.java

```diff
@@ -0,0 +1,183 @@
+package pantrypal.inventory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class IngredientInventory {
```

Comment:
For all functions in this class, the method header comments need to be of the following form:
> From java coding standard: In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... etc. (not Return or Returning etc.). 

It is also better if you write them JavaDoc style.